src/Service/RabbitMQQueueStatusGuzzleProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NotificationBundle\Service;
+
+use Guzzle\Http\Client as GuzzleClient;
+use Sonata\NotificationBundle\Exception\MonitoringException;
+
+/**
+ * @deprecated Created for BC. Use Sonata\NotificationBundle\Service\RabbitMQQueueStatusHttpProvider instead
```

Comment:
@greg0ire can you propose a better message here?

Reply:
What's wrong with this one?

Reply:
I thought we were using something like: `@deprecagted since {version} ...bla blub`